README.md

```diff
@@ -161,6 +161,7 @@ The following feature flags are present in one or more of the crates listed abov
 | Feature  | Target(s)         | Implies    | Description                                                                           | Crates                                      |
 | -------- | ----------------- | ---------- | ------------------------------------------------------------------------------------- | ------------------------------------------- |
 | cuda     |                   | prove, std | Turns on CUDA GPU acceleration for the prover. Requires CUDA toolkit to be installed. | risc0-circuit-rv32im, risc0-zkp, risc0-zkvm |
+| disable-dev-mode | all | | Disables dev mode so that the proving stage may not be skipped. For the verifier, enforces that dev mode receipts generated sans proving stage are invalid. | risc0-zkvm |
```

Comment:
I suggest we use the term "fake receipts" 
```suggestion
| disable-dev-mode | all | | Disables dev mode so that the proving stage may not be skipped. For the verifier, enforces that [fake receipts](https://docs.rs/risc0-zkvm/0.18.0/risc0_zkvm/enum.InnerReceipt.html#variant.Fake) are invalid. Used to prevent a misplaced `RISC0_DEV_MODE` from breaking security in production systems. | risc0-zkvm |
```